common/lib/capa/capa/tests/test_inputtypes.py

```diff
@@ -16,27 +16,27 @@
 - test funny xml chars -- should never get xml parse error if things are escaped properly.
 
 """
-from collections import OrderedDict
 import json
-from lxml import etree
-from lxml.html import fromstring
-import unittest
 import textwrap
+import unittest
 import xml.sax.saxutils as saxutils
+from collections import OrderedDict
 
-from capa.tests.helpers import test_capa_system
 from capa import inputtypes
 from capa.checker import DemoSystem
+from capa.tests.helpers import test_capa_system
+from capa.xqueue_interface import XQUEUE_TIMEOUT
+from lxml import etree
+from lxml.html import fromstring
 from mock import ANY, patch
+from openedx.core.djangolib.markup import HTML
 from pyparsing import ParseException
 
-from capa.xqueue_interface import XQUEUE_TIMEOUT
-
 # just a handy shortcut
 lookup_tag = inputtypes.registry.get_class_for_tag
 
 
-DESCRIBEDBY = 'aria-describedby="desc-1 desc-2"'
+DESCRIBEDBY = 'aria-describedby="status_{status_id} desc-1 desc-2"'
```

Comment:
Are you referring to the following?
```
DESCRIBEDBY = 'aria-describedby="status_{status_id} desc-1 desc-2"'
HTML(DESCRIBEDBY).format(status_id=prob_id)
```
Wouldn't this be equivalent, or am I missing something?
```
DESCRIBEDBY = HTML('aria-describedby="status_{status_id} desc-1 desc-2"')
DESCRIBEDBY.format(status_id=prob_id)
```